x/epochs/handler.go

```diff
@@ -13,8 +13,6 @@ import (
 // NewHandler returns a handler for epochs module messages.
 func NewHandler(k keeper.Keeper) sdk.Handler {
 	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
-		ctx = ctx.WithEventManager(sdk.NewEventManager())
```

Comment:
IMO we shouldn't even have these `handler.go` files -- it's all legacy code.

Reply:
I'm guessing this: https://github.com/cosmos/cosmos-sdk/pull/9650 would be the right way to do it?